src/api/stripe.js

```diff
@@ -0,0 +1,59 @@
+import { BaseApi } from './base';
+import { http } from '../utils/http';
+
+
+/**
+ * @class StripeApi
+ * @extends BaseApi
+ */
+export class StripeApi extends BaseApi {
+
+  updateCustomer(userId, token) {
+    if (!token) {
+      return Promise.reject(new Error('Must a Stripe token.'))
+    }
+
+    const url = this.getFullURL('appUsers', userId, 'stripe', 'customer');
+    return this.validateAuthHeaders().then((headers) => {
+      return http('POST', url, {
+        token
+      }, headers);
+    });
+  }
+
+  checkout(userId, actionId, token) {
```

Comment:
`/checkout` isn't supposed to be part of the public API, it's used exclusively by the SMS checkout page